### DISCUSSION TITLE
The LTMEAT got patched by our school.

### DISCUSSION THREAD

**Discussion by BenThmpsn** [3kh0/ext-remover#693] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
can you please fix it.

**Comment by ghost** [comment]
same

**Reply by rwg100** [reply]
sounds like a skill issue

**Reply by ghost** [reply]
they blocked javascript and no it's does not sound like a skill issue

**Reply by rwg100** [reply]
it is simply an issue of skill

**Reply by 3kh0** [reply]
facts

**Reply by ghost** [reply]
really just shut up then

**Comment by obkduishinijf** [comment]
gives no context, explanation or anything, just says that it's broken and "can you please fix it."
if there's no explanation or details, nobody's got any idea as to what's broken.

**Reply by BenThmpsn** [reply]
How the hell am I supposed to know what's broken or not?

**Reply by obkduishinijf** [reply]
if YOU say it's broken but give no context, screenshots etc. then we have NO FUCKING CLUE what's broken.

**Reply by bartholemuew** [reply]
why you dumb

**Reply by obkduishinijf** [reply]
"why you dumb" if you're talking to me, then go ahead and explain why or HOW "it's not working."

**Comment by cinnamongold** [comment]
If it doesn't work, then your admin has most likely disabled javascript bookmarklets. Unfortunately, there is _no_ patch for this.

**Reply by cinnamongold** [reply]
You can also downgrade, but in many cases that could lead to loss of electronic privileges, or in some extreme cases, expulsion. The punishment **will** get worse as you get older (and into high school)

**Reply by AshtonDavies** [reply]
> If it doesn't work, then your admin has most likely disabled javascript bookmarklets. Unfortunately, there is _no_ patch for this.

There is a workaround. A major step to posting an issue is searching for the case if it has already been posted, and it has. Don't be lazy, and actually check the closed discussions tab, kids.

**Comment by PSYclownYT** [comment]
> can you please fix it.

SOLUTION:
go to the chrome extension's page, and use the ID to go to chrome-extension://**(ID from chrome://extensions)**/manifest.json

**Comment by Ender-Ghost-Craft** [comment]
is it because javascript: links are blocked?
when that happens you press the bookmarklet nothing happens

**Reply by obkduishinijf** [reply]
well they never gave any context, or even a description of how it's broken, so we have no idea.

**Reply by Ender-Ghost-Craft** [reply]
thanks bro

**Reply by OmarYAhme** [reply]
Hey, could you tell me how to do this thing ur talking about?

**Reply by TheGooferFox** [reply]
Dude do urself a favor and give up and just use a unblocked browser or smth lol ur wasting ur time

**Reply by OmarYAhme** [reply]
lol i just saw this

**Reply by OmarYAhme** [reply]
yeah omarahh is user

**Reply by OmarYAhme** [reply]
how

**Reply by OmarYAhme** [reply]
hurry i gotta go in 15 mins

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
ok lol

**Reply by OmarYAhme** [reply]
can you show me how to do it quick please

**Reply by OmarYAhme** [reply]
yes

**Reply by OmarYAhme** [reply]
ok show

**Reply by OmarYAhme** [reply]
dextensify?

**Reply by OmarYAhme** [reply]
hurry bro i got to go soon

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
can you please hurry up

**Reply by OmarYAhme** [reply]
bro

**Reply by OmarYAhme** [reply]
ok then show me

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
i will try

**Reply by OmarYAhme** [reply]
https://dextensify.ading.dev

**Reply by OmarYAhme** [reply]
glitch is blocked

**Reply by OmarYAhme** [reply]
glitch is blocked

**Reply by OmarYAhme** [reply]
yes

**Reply by OmarYAhme** [reply]
yeah sure

**Reply by OmarYAhme** [reply]
glitch.me is blocked in my school

**Reply by OmarYAhme** [reply]
securly blocked

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
nice

**Reply by OmarYAhme** [reply]
ok lol

**Reply by OmarYAhme** [reply]
ok but how i do unblock everything, the glitch link it blocked replit would work

**Reply by OmarYAhme** [reply]
im saying how do i do the LTMEAT 115+

**Reply by OmarYAhme** [reply]
idk how

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
nice

**Reply by OmarYAhme** [reply]
ok

**Reply by OmarYAhme** [reply]
why are you on tor bro 💀

**Reply by OmarYAhme** [reply]
im closing thee link

**Reply by OmarYAhme** [reply]
no thanks

**Reply by OmarYAhme** [reply]
i left

**Reply by OmarYAhme** [reply]
cool

**Reply by OmarYAhme** [reply]
i gtg i be back in hour 30

**Reply by Ender-Ghost-Craft** [reply]
sorry, i have school when you are online, can you turn it back on? keep it on for the day, I will respond after you turn it on

**Reply by Ender-Ghost-Craft** [reply]
Can u leave it on for a couple more hours, I'm in a car

**Reply by Ender-Ghost-Craft** [reply]
turn it on please

**Reply by obkduishinijf** [reply]
171 replies im crying

**Comment by Ender-Ghost-Craft** [comment]
Can you keep it in for a couple more hours?

**Comment by Ender-Ghost-Craft** [comment]
I'm in a car

**Comment by 3kh0** [comment]
no

**Comment by Ender-Ghost-Craft** [comment]
What Abt now

**Comment by YeeticusPrimed** [comment]


**Reply by YeeticusPrimed** [reply]
they probs blocked it in chrome://policy/    urlblocklist

**Reply by AshtonDavies** [reply]
> what about google sites version of ltmeat without url

Where might thou attain this sorcery?

**Comment by 1coolmatt8083** [comment]
can someone help me im triying to play roblox on a school crome book with lightspeed agent installed

**Reply by 1coolmatt8083** [reply]
and I cant disable it

**Reply by Y4hu4H** [reply]
did u already use the ltmeat flood method?
